runtime.go

```diff
@@ -27,6 +33,11 @@ type Runtime interface {
 	LookupBuiltinType(pkg, name string) (semantic.MonoType, error)
 }
 
+type ASTHandle interface {
+	ASTHandle()
```

Comment:
Is this a method just to ensure its intentional to implement the interface? Is it ever called? If not can you document that as its not clear what else the function would do.